apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -738,42 +708,39 @@ function doEdits(data) {
     editGet(data.url, false, false, data.contentsA, null);
     // (A, A, A, A)
 
-    badPost('save_and_sync', data.contentsB, data.url);
-    // (A, A, A, A)
-
     editPost('save_and_sync', data.contentsB, data.url, true, false, null);
+    waitForJobSequence(locals, 'Success');
     // (B, B, A, B)
 
-    waitForJobSequence(locals, 'Success');
     pullAndVerifyFileInDev(data.path, data.contentsB);
     // (B, B, B, B)
 
     editGet(data.url, false, false, data.contentsB, null);
     // (B, B, B, B)
 
     writeAndCommitFileInLive(data.path, data.contentsA);
-    // (B, A, B, B)
+    // (A, A, B, B)
 
     editGet(data.url, false, false, data.contentsA, null);
     // (A, A, B, B)
 
     writeAndCommitFileInLive(data.path, data.contentsB);
-    // (A, B, B, B)
+    // (B, B, B, B)
```

Comment:
Ditto for the above comment.

Reply:
This should be reverted for the same reason as here: https://github.com/PrairieLearn/PrairieLearn/pull/8680#discussion_r1511978340

Reply:
Updated in 82ca14b.